runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -4278,6 +4278,9 @@ void JitShutdown(J9JITConfig * jitConfig)
    // so the fact that this option is true doesn't mean that IProfiler structures were not allocated
    if (options /* && !options->getOption(TR_DisableInterpreterProfiling) */ && iProfiler)
       {
+      //if (options->getOption(TR_StoreIPInfoOnShutdown))
+      //   iProfiler->persistAllEntries();
+
```

Comment:
Does this need to be in this PR?

Reply:
Yes, it serves as an example on how to persist IProfiler at shutdown.
Actually, I plan to uncomment the lines once I get that `TR_StoreIPInfoOnShutdown` option integrated in omr.